atlp-rwanda/elite-bn-fe
# ch(configure-storybook): Configure storybook for component development

#### What does this PR do?

Configure the storybook tool to make working on related components more efficient, easier to reuse, and have better component documentation.

#### How should this be manually tested?

1. Clone branch, install packages by running npm i or yarn install.
2. Run npm start or yarn start to start the server.
3. Now you can see a page with the Barefoot Nomad title opened in your browser.
4. run test
5. run storybook

#### What are the relevant pivotal tracker stories?
[#182201523](https://www.pivotaltracker.com/story/show/182201523)
